### DISCUSSION TITLE
qutebrowser searches the path of file if it can't find it in filesystem

### DISCUSSION THREAD

**Discussion by alandiwix** [qutebrowser/qutebrowser#7308] [Q&A] [answered]
qutebrowser searches the path of file if it can't find it in filesystem using default search engine which could lead to serious privacy issues. Is there any way to disable this behaviour?

**Comment by The-Compiler** [comment]
You can set `url.auto_search` to e.g. `never`. With it enabled, how would qutebrowser know whether what you're entering is a (non-existent) file path and not a search term?

(also, FWIW, your search engine probably knows a lot more interesting stuff about you privacy-wise than the path of a file)

**Reply by alandiwix** [reply]
Well, first it tries to search for the file in a file system and doesn't find it, right? Could it just give an error like for non-existing domain name instead of trying to google it? I understand that this is different mechanism, but logially non-existing website path should have the same behaviour as non-existing file.

**Reply by The-Compiler** [reply]
That is the behavior of setting `auto_search` to `never`. By default, `:open` can be used to open URLs, searches, *and* local files (plus some other stuff, like quickmarks). Thus, if you input, say `/tmp/doesnotexist`, how would qutebrowser know that's a file path, but, say, `/e/os` is [something](https://e.foundation/e-os/) you want to search for?

**Reply by alandiwix** [reply]
@The-Compiler Ok, I found the right thing to do - specify the protocol before:
```
file:///path/to/file
```
in this case it works as expected

**Comment by alandiwix** [comment, marked as answer]
I should specify the `file://` before the filepath
```
qutebrowser file:///path/to/file
```
